Makefile

```diff
@@ -36,7 +52,15 @@ legato: cert
 	#        which can not be downloaded when the 'fetch' option is used.
 	bazel --output_base=$(OUTPUT_BASE_DIR) build //endpoint:libendpoint.so
 	# Generate endpoint Legato app
-	(cd endpoint && leaf shell -c "mkapp -v -t wp77xx endpoint.adef")
+    ifeq ($(TARGET), development_host)
```

Comment:
Follow the consistent indention.

Reply:
It must be space, not tab.

Reply:
By using define directives, you can customize commands which would be placed around here.
Check this: http://www.chiark.greenend.org.uk/doc/make-doc/make.html/Using-Variables.html

Reply:
For example, F9 microkernel invokes platform-specific flash command in flexible ways: https://github.com/f9micro/f9-kernel/blob/master/mk/generic.mk#L79

Reply:
Always write down elegant and pretty code.